src/js/vendors/flickity.js

```diff
@@ -9,5 +10,4305 @@
+	}
+
+	function noop() { }
+
+	const logError = typeof console === 'undefined' ? noop :
+		function( message ) {
+			// eslint-disable-next-line no-console
+			console.error( message );
+		};
+
+	// -------------------------- measurements -------------------------- //
+
+	const measurements = [
+		'paddingLeft',
+		'paddingRight',
+		'paddingTop',
+		'paddingBottom',
+		'marginLeft',
+		'marginRight',
+		'marginTop',
+		'marginBottom',
+		'borderLeftWidth',
+		'borderRightWidth',
+		'borderTopWidth',
+		'borderBottomWidth',
+	];
+
+	const measurementsLength = measurements.length;
+
+	function getZeroSize() {
+		const size = {
+			width: 0,
+			height: 0,
+			innerWidth: 0,
+			innerHeight: 0,
+			outerWidth: 0,
+			outerHeight: 0,
+		};
+		for ( let i = 0; i < measurementsLength; i++ ) {
+			const measurement = measurements[ i ];
+			size[ measurement ] = 0;
+		}
+		return size;
+	}
+
+	// -------------------------- getStyle -------------------------- //
+	function getStyle( elem ) {
+		// eslint-disable-next-line no-undef
+		const style = getComputedStyle( elem );
+		if ( ! style ) {
+			logError( 'Style returned ' + style +
+                '. Are you running this code in a hidden iframe on Firefox? ' +
+                'See https://bit.ly/getsizebug1' );
+		}
+		return style;
+	}
+
+	// -------------------------- setup -------------------------- //
+
+	let isSetup = false;
+
+	let isBoxSizeOuter;
+
+	/**
+   * setup
+   * check isBoxSizerOuter
+   * do on first getSize() rather than on page load for Firefox bug
+   */
+	function setup() {
+		// setup once
+		if ( isSetup ) {
+			return;
+		}
+		isSetup = true;
+
+		// -------------------------- box sizing -------------------------- //
+
+		/**
+     * Chrome & Safari measure the outer-width on style.width on border-box elems
+     * IE11 & Firefox<29 measures the inner-width
+     */
+		const div = document.createElement( 'div' );
+		div.style.width = '200px';
+		div.style.padding = '1px 2px 3px 4px';
+		div.style.borderStyle = 'solid';
+		div.style.borderWidth = '1px 2px 3px 4px';
+		div.style.boxSizing = 'border-box';
+
+		const body = document.body || document.documentElement;
+		body.appendChild( div );
+		const style = getStyle( div );
+		// round value for browser zoom. desandro/masonry#928
+		isBoxSizeOuter = Math.round( getStyleSize( style.width ) ) === 200;
+		getSize.isBoxSizeOuter = isBoxSizeOuter;
+
+		body.removeChild( div );
+	}
+
+	// -------------------------- getSize -------------------------- //
+
+	function getSize( elem ) {
+		setup();
+
+		// use querySeletor if elem is string
+		if ( typeof elem === 'string' ) {
+			elem = document.querySelector( elem );
+		}
+
+		// do not proceed on non-objects
+		if ( ! elem || typeof elem !== 'object' || ! elem.nodeType ) {
+			return;
+		}
+
+		const style = getStyle( elem );
+
+		// if hidden, everything is 0
+		if ( style.display === 'none' ) {
+			return getZeroSize();
+		}
+
+		const size = {};
+		size.width = elem.offs
```

Comment:
Adding this comment for additional context. This change removes the SVG element which Flickity was using and instead uses div + button styled with the appropriate classes to achieve the correct style. Classes used are defined in lightbox utility.

https://github.com/godaddy/coblocks/blob/23c76198775064f1717d652ea71e4814c450dcfe/src/components/block-gallery/styles/style/_lightbox.scss#L64